Makefile

```diff
@@ -172,6 +172,37 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 	docker-compose -f docker-compose.yaml up haproxy
 endif
 
+run-multi: run-client
```

Comment:
Can we introduce `client`-free versions of these commands? While the "out of the box" workflow has developers running the server and client in the same terminal (competing with STDOUT), I think most of us eventually migrate to running `make run-server` and `npm run run` in separate terminals.

In the existing code, something like a `run-ha: check-prereqs run-haserver run-client` target with `run-haserver` no longer depending on `run-client`. (Not 100% sure how that translates into the new `run-multi` stuff.)

Reply:
We can defo do that. i think simpler more composable commands will serve us better, will amend.